qiling/os/windows/api.py

```diff
@@ -8,7 +8,7 @@
 # See: https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types
 
 LONG   = PARAM_INTN
-ULONG  = PARAM_INTN
+ULONG  = PARAM_INT32
```

Comment:
How come `ULONG` size is different from `LONG`'s ?
If Windows treats `LONG` as a 32-bit integer regardless of the arch native size, so both should be changed.

Reply:
I did not mean to include this change in the PR, this can be removed.